oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentStore.java

```diff
@@ -2317,10 +2320,19 @@ private <T extends Document> DocumentStoreException handleException(Throwable ex
                 invalidateCache(collection, id);
             }
         }
+
+        if (ex instanceof MongoWriteException) {
```

Comment:
Wondering if `MongoWriteException` is indeed the one we like to count.

Looking at the [MongoServerException](https://mongodb.github.io/mongo-java-driver/3.12/javadoc/com/mongodb/MongoServerException.html) it looks like there are a bulk siblings that we might also be interested : `BulkWriteException`, `MongoBulkWriteException` ?

If that's the case, then I'm wondering if we should count `MongoServerException` instead - but arguably that's less specific than the current `MongoWriteException` ...

Reply:
+1 to go ahead with this one and then consider the bulk ones later

Reply:
I verified that this is the right one; see https://issues.apache.org/jira/browse/OAK-10642?focusedCommentId=17819206&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17819206

Reply:
So proposal is to go ahead with this and expand this later on when we have more time.